capa/features/freeze.py

```diff
@@ -264,6 +264,14 @@ def main(argv=None):
     parser.add_argument(
         "-f", "--format", choices=[f[0] for f in formats], default="auto", help="Select sample format, %s" % format_help
     )
+    parser.add_argument(
+        "-b",
+        "--backend",
+        type=str,
+        help="select the backend to use in Python 3 (this option is ignored in Python 2)",
```

Comment:
we could add this option only under python3 and then keep the help text simpler

Reply:
I guess we want to do the same in `capa/main.py`, right?

Reply:
yeah, if possible